content/_data/changelogs/weekly.yml

```diff
@@ -7562,6 +7562,78 @@
 
   # pull: 4708 (PR title: Fix up Protector class)
 
+- version: '2.237'
+  date: 2020-05-18
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4712
+    issue: 61920
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent telemetry warnings about missing <code>javax.annotation</code> classes when running with Java 11 (regression in 2.231).
+  - type: bug
+    category: bug
+    pull: 4718
+    issue: 62231
+    authors:
+    - MRamonLeon
+    message: |-
+      Prevent Old Data Monitor from failing plugin loading in the case of class field unmarshalling issues.
+  - type: bug
+    category: bug
+    pull: 4711
+    issue: 60118
+    authors:
+    - oleg-nenashev
+    message: |-
+      Ensure that <code>UserLanguages</code> telemetry initializer always runs after extensions are augmented.
+  - type: bug
+    category: bug
+    pull: 4684
+    issue: 61956
+    authors:
+    - calvinpark
+    message: |-
+      Ensure that job/folder creation routines properly check the requested name for invalid characters.
+  - type: rfe
+    category: developer
+    pull: 4725
+    authors:
+    - jvz
+    message: |-
+      Developer: Update Apache Ant from 1.10.7 to 1.10.8.
+  - type: rfe
+    category: developer
+    pull: 4656
+    authors:
+    - jvz
+    references:
+      - pull: 4656
+      - url: https://tomcat.apache.org/taglibs/CHANGES.txt
+        title: Changelog up to 1.2.5
+      - url: https://github.com/eclipse-ee4j/jstl-api/compare/1.2.3-RELEASE..1.2.7-RELEASE
+        title: Diff of 1.2.3 to 1.2.7
+      - url: https://github.com/javaee/jstl-api/compare/javax.servlet.jsp.jstl-1.2.1..javax.servlet.jsp.jstl-1.2.3
+        title: Diff of 1.2.1 to 1.2.3
+    message: |-
+      Developer: Update the JSTL API library from 1.2.1 to 1.2.7.
```

Comment:
Not a `Developer:` issue either.

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/3556